news/base16-shell.rst

```diff
@@ -0,0 +1,49 @@
+**Added:**
+
+* Add alias to `base16 shell`_
+
+* Installation / Usage
+    1. To install use pip
+
+    .. code-block:: bash
+
+        python3 -m pip install xontrib-base16-shell
+
+    2. Add on ``~/.xonshrc``
+
+    .. code:: python
+        :number-lines:
+
+        $BASE16_SHELL = $HOME + "/.config/base16-shell/"
+        xontrib load base16_shell
+
+See image
+
+|Terminal|
```

Comment:
I believe that needs to be indented.

Reply:
https://github.com/xonsh/xonsh/blob/51d9f77fd2aee6706cefd079d4fd93be7ebeff0f/news/base16-shell.rst#L3

I wrote the link incorrectly